openml/datasets/functions.py

```diff
@@ -352,6 +353,88 @@ def get_dataset(dataset_id):
     return dataset
 
 
+def create_dataset(name, description, creator, contributor, collection_date,
+                   language, licence, attributes, data, default_target_attribute,
+                   row_id_attribute, ignore_attribute, citation, format="arff",
+                   original_data_url=None, paper_url=None, update_comment=None,
+                   version_label=None):
+    """Create a dataset.
+
+    This function creates an OpenMLDataset object.
+    The OpenMLDataset object contains information related to the dataset
+    and the actual data file.
+
+    Parameters
+    ----------
+    name : str
+        Name of the dataset.
+    description : str
+        Description of the dataset.
+    creator : str
+        The person who created the dataset.
+    contributor : str
+        People who contributed to the current version of the dataset.
+    collection_date : str
+        The date the data was originally collected, given by the uploader.
+    language : str
+        Language in which the data is represented.
+        Starts with 1 upper case letter, rest lower case, e.g. 'English'.
+    licence : str
+        License of the data.
+    attributes: list
+        A list of tuples. Each tuple consists of the attribute name and type.
+    data : numpy.matrix
```

Comment:
This is not correct, it should be `array-like or sparse matrix, shape=(n_samples, n_features)`